tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -61,16 +75,43 @@ def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
             c2r.sendline("y")
             prompt_amount -= 1
 
-        assert c2r.expect("Kernel modules are compatible.") == 0
+        # Stop conversion before the point of no return as we do not need to run the full conversion
+        # with the check for kompatible kernel modules being right before the PONR.
+        assert c2r.expect("All loaded kernel modules are available in RHEL.") == 0
         c2r.expect("Continue with the system conversion?")
         c2r.sendline("n")
         assert c2r.exitstatus != 0
 
 
-def test_tainted_kernel_inhibitor(shell, convert2rhel):
-    # This test marks the kernel as tainted which is not supported by convert2rhel.
+def test_inhibit_if_module_is_force_loaded(shell, convert2rhel):
+    """
+    Test force loads kmod and checks for Convert2RHEL not being inhibited.
+    With the check for tainted kernel modules being at the beginning of the script, abort the conversion ASAP
+    for the sake of test speed.
+    """
+    # Force load the kernel module
+    assert shell("modprobe -f -v bonding").returncode == 0
+    # Check for force loaded modules being flagged FE in /proc/modules
+    assert "(FE)" in shell("cat /proc/modules").output
+
+    with convert2rhel("--no-rpm-va --debug") as c2r:
+        assert c2r.expect("Tainted kernel module\(s\) detected") == 0
```

Comment:
```suggestion
        assert c2r.expect("Tainted kernel modules detected") == 0
```
So it's the same as below or whatever of these two is correct?

Reply:
I changed the log message in https://github.com/oamg/convert2rhel/commit/5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f. @r0x0d, is there a reason why you dropped the commit?

Reply:
This PR had a rebase problem where we had a lot of commits from main here... I just pushed my local changes and applied Daniel's commits on top of it.

Reply:
@bocekm do you mean that the commit is not available here anymore or the change proposed is missing?

Reply:
Both. The commit is not among the PR commits and the changes from the commit are not part of any other PR commit.

Reply:
I see. Could you please comment again the change so I can apply it?

Reply:
All the changes are already in https://github.com/oamg/convert2rhel/commit/5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f.

You can use the approach @abadger suggested on Monday when you dropped @danmyway's commits from https://github.com/oamg/convert2rhel/pull/554:
```
$ git pull --rebase
$ wget https://github.com/oamg/convert2rhel/commit/5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f.patch
$ git am 5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f.patch
```

Reply:
Okay, I applied the changes from 5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f.

Reply:
Not completely, for some reason. The map/lambda block is still present.